conda/meta.yaml

```diff
@@ -24,6 +24,7 @@ test:
     - ts-idl {{ idl_version }}
     - ts-salobj {{ salobj_version }}
     - ts-tcpip
+    - ts-dream-common
```

Comment:
you probably need to add `ts-dream-common` to the "run" requirements.. Also, I noticed `ts-tcpip` is not part of the run requirements.

Reply:
Done

Reply:
Done